.eslintrc.json

```diff
@@ -0,0 +1,34 @@
+{
+  "root": true,
+  "parser": "@typescript-eslint/parser",
+  "plugins": ["simple-import-sort"],
+  "extends": ["airbnb-base", "airbnb-typescript/base", "prettier"],
+  "parserOptions": {
+    "project": "./tsconfig.json"
+  },
+  "settings": {
+    "import/resolver": {
+      "typescript": {}
+    }
+  },
+  "rules": {
+    "simple-import-sort/imports": [
+      "error",
+      {
+        "groups": [["^@solana", "^@?\\w"], ["^[^.]"]]
+      }
+    ],
+    "import/prefer-default-export": "off",
+    "no-restricted-imports": [
+      "error",
+      {
+        "patterns": [".*"]
+      }
+    ],
+    "@typescript-eslint/no-use-before-define": "warn",
+    "no-plusplus": "off",
+    "import/no-cycle": "warn",
+    "no-await-in-loop": "off",
```

Comment:
think we can keep this on and just use eslint disable comments when we need to break the rule like what you did in signAndSendTransactionSequence?

Reply:
Agreed, will update